### ISSUE TITLE
PS3 Slim crashes.

### ISSUE THREAD

**Issue by Mikeykorby** [aldostools/webMAN-MOD#1044]
Whenever I launch an app with my PS3 Slim Hybrid Firmware 4.91* With hen and Webman installed, as soon I open a game, it crashes. Like not even freeze, it just full on turns off my Playstation.

**Comment by aldostools** [comment]
4.90 has a LV2 kernel from 4.80. I recommend you upgrade to HFW 4.91 + HEN 3.3.0 or later.

Previous versions of webMAN MOD may not work on HFW 4.90, but they work on HFW 4.91.

**Comment by Mikeykorby** [comment]
srry mistake i mean 4.91 :)

**Comment by Mikeykorby** [comment]
thx though

**Comment by Mikeykorby** [comment]
any other suggestions?

**Comment by Mikeykorby** [comment]
Proof btw: Compressed

**Comment by Mikeykorby** [comment]
Uploading IMG_0345[1] (1) (1) (1).mp4…

**Comment by Mikeykorby** [comment]
https://github.com/aldostools/webMAN-MOD/assets/153252533/c3e78635-13cc-450a-b35b-3c478344d280

**Comment by Mikeykorby** [comment]
anyone came up with a solution yet?

**Comment by aldostools** [comment]
In your video it says HEN "developer mode"... Have you tried re-installing the regular HEN from https://ps3xploit.me?

The issue seems to be caused by the HEN payload. The game that you launched looks like a PSN version of GT6.
PSN games launch directly from XMB and they don't interact with webMAN MOD in any way.

**Comment by Mikeykorby** [comment]
Yes, I have reinstalled it multiple times before asking for a solution.
